torchbiggraph/converters/import_from_tsv.py

```diff
@@ -404,15 +373,20 @@ def main():
         print(f"The edge paths provided on the command line ({opt.edge_paths}) "
               f"don't match the ones found in the config file ({edge_paths})")
 
+    if opt.edgelist_format == "tsv":
+        edgelist_reader = TSVEdgelistReader(opt.lhs_col, opt.rhs_col, opt.rel_col)
+    elif opt.edgelist_format == "parquet":
+        edgelist_reader = ParquetEdgelistReader(opt.lhs_col, opt.rhs_col, opt.rel_col)
+    else:
+        raise RuntimeError(f"Unknown edgelist format: {opt.edgelist_format}")
+
     convert_input_data(
```

Comment:
It's rather odd that to import from Parquet you need to use the `import_from_tsv` command... Can we create a `import_from_parquet` command instead? Or change the command's name?

Reply:
Yes, that's possible, although `import_from_tsv` and `import_from_parquet` are nearly identical (except for possibly the named columns thing). At the very least, if we keep one script for both we should call it `import_from_file`